cockroachcloud/cmek.md

```diff
@@ -0,0 +1,147 @@
+
+## CMEK FAQs
+<!-- TODO: De-duplicate with the standalone topic -->
+The following FAQs distill important information that is also covered in the preceding sections, as well as in the [Limitations](#limitations) section below.
+
+### How are the store key and data key stored in the cluster?
+
+If CMEK is enabled, the store key is encrypted by the CMEK key, which is never persisted to the cluster and never resides within {{ site.data.products.db }}. Otherwise, the store key is not encrypted at rest, but it is still protected by the storage-level encryption on each of the cluster's nodes.
+
+Data keys are encrypted at rest by the store key, which is protected by the CMEK key.
+
+### If CMEK is not enabled, is data at rest in a {{ site.data.products.dedicated }} cluster encrypted by default?
+
+Yes. Data at rest in a {{ site.data.products.dedicated }} cluster is encrypted in persistent storage by your IAAS provider. For more details, check the documentation for your IAAS provider.
+
+### Can the CMEK key be automatically or manually rotated?
+
+It's not possible to rotate the CMEK key.
+
+### Are managed backups in CockroachDB Cloud encrypted using the CMEK key?
+
+When CMEK is enabled, managed backups in {{ site.data.products.db }} are encrypted using the CMEK key before being written to persistent storage, and the CMEK key must be available when you restore from an automatic backup. In addition, backups taken before CMEK was enabled cannot be restored to the cluster.
+
+### Can I enable CMEK on an existing cluster?
+
+We do not recommend enabling CMEK on an existing cluster, to prevent a situation where some data is protected with the CMEK key and other data isn't. For the same reason, we recommend that you configure CMEK after creating the cluster but before loading any data into it or giving users access to it.
+
+This limitation also applies to adding a new region to an existing cluster. Adding a new region to a multi-region cluster that has CMEK enabled is not recommended, because the new region's data at rest will not be encrypted by the CMEK key.
+
+### Can I enable CMEK from Cockroach Cloud Console?
+
+It's not possible to enable CMEK from {{ site.data.products.db }} Console. Instead, you must use the [Cloud API](/docs/cockroachcloud/cloud-api.html).
+
+### After enabling CMEK, can I disable it?
+
+CMEK can't be disabled on a cluster after it has been enabled.
+
+## Limitations
+The CMEK feature has the following limitations:
+
+- CMEK can be enabled only on clusters created after April 1, 2022 (AWS) or June 9, 2022 (GCP).
+- To enable CMEK, you must use the [Cloud API](/docs/cockroachcloud/cloud-api.html). It's not possible to enable CMEK using the {{ site.data.products.db }} Console.
+- Adding a region to a cluster with CMEK enabled is not recommended because currently, the new region will not be protected by the CMEK key.
+- Rotating a CMEK key is not supported.
+- Only symmetric AES-GCM software keys are supported.
```

Comment:
I would remove this. We're not constraining anyone - it all depends on what cloud providers support. At this point it looks like we're the bottleneck.